kuznetsov/quizzes/src/main/java/com/kuznetsov/enteties/Credentials.java

```diff
@@ -0,0 +1,47 @@
+package enteties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Credentials {
```

Comment:
Better to separate savedCredentials logic into separate service, to not mix logic with entity

Reply:
Understood. Thank you.